github3/repos/branch.py

```diff
@@ -283,12 +283,35 @@ class ProtectionEnforceAdmins(models.GitHubCore):
     .. links
     .. _Branch protection API documentation:
         https://developer.github.com/v3/repos/branches/#get-branch-protection
+    .. _Admin enforcement of protected branch
+        https://developer.github.com/v3/repos/branches/#get-admin-enforcement-of-protected-branch
     """
 
     def _update_attributes(self, protection):
         self._api = protection['url']
         self.enabled = protection['enabled']
 
+    @decorators.requires_auth
+    def enable(self):
+        """Enable Admin enforcement for protected branch"""
+        resp = self._post(self._api,
+                          headers=BranchProtection.PREVIEW_HEADERS_MAP)
+        return self._boolean(resp, 200, 404)
+
+    @decorators.requires_auth
+    def disable(self):
+        """Disable Admin enforcement for protected branch"""
+        resp = self._delete(self._api,
+                            headers=BranchProtection.PREVIEW_HEADERS_MAP)
+        return self._boolean(resp, 204, 404)
+
+    @decorators.requires_auth
+    def update(self):
```

Comment:
How is this supposed to be used? Are you expecting folks to do

```py
enforce_admins.enabled = True
enforce_admins.update()
```

That's completely foreign to this library and I do not want to add that as a pattern. Please, let's not add this method if there are separate `enable` and `disable` methods (which I like and want to keep).